docs/.vitepress/config.mts

```diff
@@ -26,7 +26,7 @@ const sidebar = [
     text: 'Using Zap',
     items: [
       { text: 'Generate Code', link: '/usage/generation' },
-      { text: 'Event Usage', link: '/usage/events' },
+      { text: 'Generated API', link: '/usage/generated-api' },
```

Comment:
I'm not convinced on `Generated API`. Maybe something like `Using Zap`?

Reply:
I'm also not particularly convinced on it, but I couldn't really think of any good alternatives. `Using Zap` is the name of the parent section, so that seems even stranger. Maybe "Event and Function API"?

Reply:
Or maybe "Sending Events and Functions"

Reply:
Ok - let's do "Event and Function API" then